client/src/_shared/forms/DropdownInput.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import ExpandLessIcon from '@material-ui/icons/ExpandLess'
+
+/**
+ * Custom dropdown input including a label, that accepts styling
+ *
+ * @param {boolean}  [combo]             Indicates if the dropdown is displayed individually or as a combo box with a text input.
+ * @param {string}   [containerClasses]  Custom classes to be applied to the container div.
+ * @param {string}   [defaultOption]     The value of the default option, if one should be set.
+ * @param {string}   inputId             Unique identifier for a rendered component.
+ * @param {string}   [labelClasses]      Custom classes to be applied to the label div.
+ * @param {string}   [label]             The display text for the label div.
+ * @param {func}     onChange            Callback to be triggered when the dropdown's selected option changes.
+ * @param {string}   [optionClasses]     Custom classes to be applied to the "options" in the dropdown.
+ * @param {Object[]} options             Array of options with a value (for direct comparison) and a label (for display).
+ * @param {string}   [placeholder]       Placeholder text to display inside the dropdown select box.
+ * @param {boolean}  [required]          Indicates whether or not the dropdowns's value is required.
+ * @param {string}   [selectClasses]     Custom classes to be applied to the "select" box div.
+ *
+ */
+
+export default function DropdownInput({
```

Comment:
An issue that custom drop downs have is that they suck on mobile devices. They're hard to use in the tiny screens, and they require extra work to make them less awful. If mobile devices are an important target, this is something to really consider.

Reply:
Gonna make this a refactor ticket; right now we're not targeting mobile devices, but I'm curious about why/how they suck.  I know I'm on an iPhone, but I'd be curious what/how I should test so I can see the suckiness first hand for troubleshooting.